### ISSUE TITLE
`placeOrder` message `Unknown payload type 2142`

### ISSUE THREAD

**Issue by hesedel** [Reiryoku-Technologies/Mida-cTrader#4]

**Comment by Vasile-Peste** [comment]
> sorry, the issue with placing an order was just a matter of having a wrong `access_token` with incorrect permissions ;)

Hey, I realized this but unfortunately this is not properly handled on our side.
We will improve this by adding a more detailed error message.

Thank you!

**Comment by hesedel** [comment]
right, also when I set a `protection` prop, it also throws an error:
```
protection: {
        stopLoss: 34000,
        takeProfit: 33940,
      }

**Comment by hesedel** [comment]
got it now, that it's a matter of also not using 'MARKET', it needs to be used with 'STOP' or 'LIMIT' types
